watertap/unit_models/reverse_osmosis_base.py

```diff
@@ -737,6 +802,13 @@ def calculate_scaling_factors(self):
         if iscale.get_scaling_factor(self.recovery_vol_phase) is None:
             iscale.set_scaling_factor(self.recovery_vol_phase, 1)
 
+        if self.config.transport_model == TransportModel.SKK:
```

Comment:
Could also handle alpha and reflect_coeff by checking they exist with `hasattr`, but this works too.